convert.go

```diff
@@ -320,6 +320,29 @@ func unquoteIfPossible(s string) (string, error) {
 	return strconv.Unquote(s)
 }
 
+func wrapParagraph(s string, l int, prefix string) string {
+	var ret string
+
+	s = strings.TrimSpace(s)
+	paragraphs := strings.Split(s, "\n\n")
+
+	for i := range paragraphs {
+		s = strings.TrimSpace(paragraphs[i])
+
+		if i != 0 {
+			ret += prefix
+		}
+
+		ret += wrapText(s, l, prefix)
+
+		if i+1 < len(paragraphs) {
```

Comment:
This is one thing that always bugs me when I see it. "Change the constant not the variable" as in `i < len(...)-1` is better than `i+1 < len(...)`. A compiler can easier see that `len(...)-1` is just another constant and will do the subtraction once (if possible) instead of N additions for `i+1`. Unless of course this particular optimization is implemented.